examples/tennis/Tennis.cpp

```diff
@@ -259,7 +267,7 @@ int main()
         }
 
         // Clear the window
-        window.clear(sf::Color(50, 200, 50));
+        window.clear(sf::Color(50, 50, 50));
```

Comment:
Why the color change?

Reply:
To answer all three questions about the colour: I got a little carried away with the enhancements. My changes should have really been in two PRs. One for the copyright stuff, one for these possible enhancements. I might properly remove these and squish da commit.

Reply:
I don't really mind if you change stuff slightly, but it's probably best if you mention it in the PR description, so it's clear that it was intentional. 🙂